nncf/telemetry_wrapper/telemetry.py

```diff
@@ -0,0 +1,49 @@
+"""
+ Copyright (c) 2022 Intel Corporation
```

Comment:
I don't like the name `_wrapper` in the name of the module. You can call it just `telemetry/telemetry.py` and define the module content in the `__init__.py` so that the import will look like `from nncf.telemetry import NNCFTelemetry`.

Reply:
We do not expect users to utilize this. Nevertheless, I refactored the package names so that the effect is the same.